### ISSUE TITLE
Prompting me to create a new account

### ISSUE THREAD

**Issue by Lebofly** [Lavoaster/IPS-SteamLoginMethod#114]
https://i.imgur.com/fg2z89i.png

Not sure if I've done something incorrectly but it's not recognizing the old steam accounts rather making new ones

**Comment by Lavoaster** [comment]
Check the ACP there should be a background task for migrating login data.

**Comment by Lebofly** [comment]
Can't seem to find that, I also forced the queue to run

**Comment by QUT-Michael-L** [comment]
Yeah even with the background task it still ignores the already defined user. If you look at the core_members table, it just creates another user with the same steamid.

**Comment by LavaToaster** [comment]
@MikeyMatou Is it still running, or had it finished? It's plausible that would happen before a member has been migrated to the 4.3 style of login links.

**Comment by QUT-Michael-L** [comment]
Wait to confirm, are you referring to the profilesync task under the Tasks section?

**Comment by Lebofly** [comment]
regardless all tasks have been run since I installed it, like Mikey said it's merely creating a new users with the same SteamID

**Comment by QUT-Michael-L** [comment]
Just an uh...quick bump. If you have any ideas it would help, because no user can login, only re-create an account with the exact same data.

**Comment by jrod9k1** [comment]
Just ran into a similar issue myself after upgrading. Go to your "core_login_methods" table. Find the method for steam logins and get the "login_id", in my case it was "11". Note that ID down and go to your "core_login_links" and for each user there will be a "token_login_method" column with an ID that should match your "login_id" for the steam method. If it does not like in my case you will need to update the ID for your users.

Something like `UPDATE core_login_links SET token_login_method=11 WHERE token_login_method=10` should do the trick. Assuming 10 is your old steam method ID of course. Obv be careful running SQL on a prod database and back your shit up before doing so

**Comment by itsAliAvali** [comment]
Depending on everyones setup that number will averagely be different.

**Comment by QUT-Michael-L** [comment]
Unfortunately, that did not fix it, but I've realised the table has been wiped, so that pretty much sums up the issue. Thanks.
